sql/_project_record_ids.sql

```diff
@@ -27,26 +27,30 @@ dbscan AS (
 project_record_join AS (
 	SELECT 
 	    a.record_id,
-	    COUNT(record_id) OVER(PARTITION BY id) as records_in_project,
+	    COUNT(record_id) OVER(PARTITION BY id) AS records_in_project,
 	    a.id,
-	    (a.geom IS NULL)::integer as no_geom,
+	    (a.geom IS NULL)::integer AS no_geom,
 	    a.geom
 	FROM dbscan a
 ),
 all_intersections AS (
-	SELECT 
-		ST_UNION(ST_INTERSECTION(a.geom, b.geom)) as intersect_geom
+	SELECT
+		ST_UNION(ST_INTERSECTION(a.geom, b.geom)) AS intersect_geom,
+		a.id
 	FROM  project_record_join a, project_record_join b
 	WHERE a.record_id < b.record_id
 	AND a.records_in_project > 1
 	AND b.records_in_project > 1
 	AND a.no_geom = 0
 	AND b.no_geom = 0
+	AND a.id = b.id
+	GROUP BY a.id
 )
 SELECT
-	array_agg(a.record_id) as project_record_ids
-INTO _project_record_ids
+	array_agg(a.record_id) AS project_record_ids
 FROM project_record_join a, all_intersections b
-WHERE ST_OVERLAPS(a.geom, b.intersect_geom)
+WHERE (ST_Overlaps(a.geom, b.intersect_geom) OR ST_Contains(a.geom, b.intersect_geom))
 AND a.id IS NOT NULL
-GROUP BY a.id;
\ No newline at end of file
+AND a.id = b.id
+GROUP BY a.id
+;
```

Comment:
To stop this table from printing to the console, this statement needs to be surrounded by parenthesis